### ISSUE TITLE
UI does not show error message if an error occurs during installation of extension

### ISSUE THREAD

**Issue by IsaSih** [rancher-sandbox/rancher-desktop#4497]
### Actual Behavior

Tried to install Epinio in Kubernetes with it disabled. UI freezes instead of displaying an error message.

### Steps to Reproduce

1. Fresh installed Rancher Desktop with Kubernetes disabled
2. Installed Epinio from the extensions catalog
3. Tried to install Epinio in the cluster

### Result

![epinio-windows](https://user-images.githubusercontent.com/70187881/232929483-10885599-0227-4881-b326-bdfa9b366ce9.gif)

UI just remains unresponsive and does throw an error message.

### Expected Behavior

Once something goes wrong with the installation, an error message should be displayed.

### Rancher Desktop Version

 1.7.0-1049-g4fc5da75

### Rancher Desktop K8s Version

N/A

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

windows 11

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by Nino-K** [comment]
Isn't this an expected behavior since k8s is disabled? but maybe a warning would clarify the confusion.

**Comment by jandubois** [comment]
> Isn't this an expected behavior since k8s is disabled? but maybe a warning would clarify the confusion.

Yes, it is, but the extension would normally display a warning when it can't get the node name, or the node name isn't `docker-desktop`. I've modified our fork to just check that the current context matches `*-desktop`, which unfortunately doesn't check that the cluster is running.

This will be resolved upstream, as our fork is supposed to be short-lived.
